packages/core/src/shipping/shipping-country-request-sender.ts

```diff
@@ -6,8 +6,12 @@ import { CountryResponseBody } from '../geography';
 export default class ShippingCountryRequestSender {
     constructor(private _requestSender: RequestSender, private _config: { locale?: string }) {}
 
-    loadCountries({ timeout }: RequestOptions = {}): Promise<Response<CountryResponseBody>> {
-        const url = '/internalapi/v1/shipping/countries';
+    loadCountries(
+        channelId: number,
+        { timeout }: RequestOptions = {},
+    ): Promise<Response<CountryResponseBody>> {
+        const url = `/internalapi/v1/shipping/countries?channel_id=${channelId}`;
```

Comment:
To double check on this, we'll _always_ have a Channel ID?

Reply:
Yeah apparently, based on what Typescript is telling me

When I first load the [checkout-store](https://github.com/bigcommerce/checkout-sdk-js/pull/2579/files#diff-6a7b4a99bde46e7576bff595ea435584d8c920521ea217e9e913f6447b2179a6R18) it will load a definite [checkout](https://github.com/bigcommerce/checkout-sdk-js/blob/8c95f363d28bac904ac2b19db5096a5b4f37ee7b/packages/core/src/checkout/checkout-selector.ts#L17) or throw an exception.

And inside the [checkout](https://github.com/bigcommerce/checkout-sdk-js/blob/8c95f363d28bac904ac2b19db5096a5b4f37ee7b/packages/core/src/checkout/checkout.ts#L44) is a [mandatory channelId](https://github.com/bigcommerce/checkout-sdk-js/blob/8c95f363d28bac904ac2b19db5096a5b4f37ee7b/packages/core/src/checkout/checkout.ts#L44). It has to exist